packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -92,6 +92,9 @@ export const enableNewReconciler = __VARIANT__;
 
 export const enableRecursiveCommitTraversal = false;
 
+// WWW is on the new sync behavior by default unless overridden by roots.
+export const enableSyncDefaultUpdates = __VARIANT__;
```

Comment:
This should re-export the dynamic flag

Reply:
Don't we want to force this off for www unless the root is overridden?

Reply:
We can do that by setting the flag in the www ReactFeatureFlags module. As written, this will hardcode the flag to `true` in the forked build.

Reply:
Isn't this the www ReactFeatureFlags file? I'm doing the same thing here as `enableNewReconciler` is above, does that have special casing somewhere?